diff --git a/cmake/config.cmake b/cmake/config.cmake
index b9a3aaef7..b9b8b9e07 100644
--- a/cmake/config.cmake
+++ b/cmake/config.cmake
@@ -68,7 +68,7 @@ set(USE_AOCL OFF)
 # - ON: enable OpenCL with cmake's auto search
 # - OFF: disable OpenCL
 # - /path/to/opencl-sdk: use specific path to opencl-sdk
-set(USE_OPENCL OFF)
+set(USE_OPENCL ON)
 
 # Whether enable Metal runtime
 set(USE_METAL OFF)
@@ -79,7 +79,7 @@ set(USE_METAL OFF)
 # - ON: enable Vulkan with cmake's auto search
 # - OFF: disable vulkan
 # - /path/to/vulkan-sdk: use specific path to vulkan-sdk
-set(USE_VULKAN OFF)
+set(USE_VULKAN ON)
 
 # Whether to use spirv-tools.and SPIRV-Headers from Khronos github or gitlab.
 #
@@ -148,7 +148,7 @@ set(USE_LLVM OFF)
 set(USE_BYODT_POSIT OFF)
 
 # Whether use BLAS, choices: openblas, atlas, apple
-set(USE_BLAS none)
+set(USE_BLAS openblas)
 
 # Whether to use MKL
 # Possible values:
@@ -367,7 +367,7 @@ set(USE_PAPI OFF)
 #   be enabled, otherwise it will be disabled.
 # Note that cmake will use `find_package` to find GTest. Please use cmake's
 # predefined variables to specify the path to the GTest package if needed.
-set(USE_GTEST AUTO)
+set(USE_GTEST OFF)
 
 # Enable using CUTLASS as a BYOC backend
 # Need to have USE_CUDA=ON
